config/env_sample.json

```diff
@@ -117,6 +117,53 @@
             }
         ]
     },
+    "LTI_CONFIG_TEMPLATE": {
+        "title": "MyLA autoconfig-{timestamp}",
```

Comment:
If we don't specify the `KeyName` in the LTI dev key, will this value here replaced their?

Reply:
I got confused with KeyName and title.